src/v2/Components/ArtistInsightsBadges.tsx

```diff
@@ -0,0 +1,150 @@
+import { BlueChipIcon, Box, Flex, Join, Spacer, Text } from "@artsy/palette"
+import { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { extractNodes } from "v2/Utils/extractNodes"
+import { ArtistInsightsBadges_artist } from "v2/__generated__/ArtistInsightsBadges_artist.graphql"
+
+interface ArtistInsightsBadgesProps {
+  artist: ArtistInsightsBadges_artist
+}
+
+const ARTIST_BADGES_TEMPLATE = {
+  ACTIVE_SECONDARY_MARKET: {
+    title: "Active Secondary Market",
+    description: "Recent auction results in the Artsy Price Database.",
+  },
+  BLUE_CHIP_REPRESENTATION: {
+    title: "Blue Chip Representation",
+    description: "Represented by internationally reputable galleries.",
+  },
+  CRITICALLY_ACCLAIMED: {
+    title: "Critically Acclaimed",
+    description: "Recognized by major institutions and publications.",
+  },
+  ARTSY_VANGUARD: {
+    title: "The Artsy Vanguard",
+    description:
+      "Featured in Artsy's annual list of the most promising artists working today.",
+  },
+}
+
+interface ArtistBadgeProps {
+  badgeType: string
+}
+
+export const ArtistBadge: FC<ArtistBadgeProps> = ({ badgeType }) => {
+  const title = ARTIST_BADGES_TEMPLATE[badgeType].title
+  const description = ARTIST_BADGES_TEMPLATE[badgeType].description
+
+  return (
+    <Flex flexDirection="row">
+      <BlueChipIcon mr={3} fill="blue100" />
+      <Flex flexDirection="column">
+        <Text variant="sm" color="blue100">
+          {title}
+        </Text>
+        <Text color="black60">{description}</Text>
+      </Flex>
+    </Flex>
+  )
+}
+
+interface HighAuctionRecordBadgeProps {
+  highAuctionResults: object
+}
+
+export const HighAuctionRecordBadge: FC<HighAuctionRecordBadgeProps> = ({
+  highAuctionResults,
+}) => {
+  // const highAuctionRecord = `${highAuctionResults.price_realized.display}, ${highAuctionResults.organization}, ${highAuctionResults.sale_date}`
+
+  return (
+    <Flex flexDirection="row">
+      <BlueChipIcon mr={3} fill="blue100" />
+      <Flex flexDirection="column">
+        <Text variant="sm" color="blue100">
+          High Auction Record
+        </Text>
+        {/* <Text color="black60">{highAuctionRecord}</Text> */}
+      </Flex>
+    </Flex>
+  )
+}
+
+export const ArtistInsightsBadges: FC<ArtistInsightsBadgesProps> = ({
+  artist,
+}) => {
+  if (
+    !artist.insights ||
+    !artist.artistHighlights ||
+    !artist.auctionResultsConnection
+  )
+    return null
```

Comment:
This is a super minor nit, but it does make the code less error prone: For single-line conditional returns its totally fine to omit brackets, eg 
```tsx
if (!foo) return null
```
but as soon as things are multiline as above, brackets should be added:
```tsx
if (...) {
  return null
}
```
Another dev without prettier could come along and accidentally add a line break between the `if` and the return and be wondering whats going on; meaning, it's prone to subtle errors.

Reply:
and its totally ok to have these many conditional checks for data existing right? just making sure my code is sound (after I've updated to multiline conditional checks)

Reply:
Oh yeah totally fine! Its all very readable.